app.rb

```diff
@@ -0,0 +1,122 @@
+require './nameable'
+require './decorator'
+require './capitalize_decorator'
+require './trimmer_decorator'
+require './book'
+require './rental'
+require './student'
+require './teacher'
+require './person'
+require './classroom'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_books
+    if @books.empty?
+      puts 'No Books records'
+      nil
+    else
+      @books.map { |book| puts "Title : \"#{book.title}\", Author: \"#{book.author}" }
+    end
+  end
+
+  def list_people
+    if @people.empty?
+      puts 'No People records'
+      nil
+    else
+      @people.map { |pupil| puts "#{pupil.class} Name: \"#{pupil.name}\", ID: \"#{pupil.id}\", Age: \"#{pupil.age}\"" }
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]:'
+    get_option = gets.chomp
+    case get_option
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    when '3'
+      puts 'wrong choice'
+    end
```

Comment:
[OPTIONAL]

- Instead of writing '3' as a wrong choice, I think it will be better if you write a condition that will output the wrong input if the user enters any other number apart from '1' and '2'

i.e You can use an `if` statement 😄  

e.g

 ```
 def create_person
    print "Do you want to create a student (1) or a teacher (2)? [Input the number]: "
    person_type = gets.chomp
    if person_type != "1" && person_type != "2"
      puts "Invalid option"
      return
    end
```